README.md

```diff
@@ -228,6 +228,43 @@ try {
 ```
 Note that in the closure-based example above, exceptions are intercepted automatically.
 
+### Oversampling Mitigation
+Oversampling mitigation allows you to ignore a parent segment/subsegment's sampled flag and instead set it to false.
+The code below demonstrates overriding the sampled flag based on the SQS messages sent to Lambda.
```

Comment:
I think this can be reworded to explain what is happening a little more clearly. 

"The code below demonstrates creating a sampled or un-sampled subsegment based on the sampling status of each SQS message processed by Lambda"
^^ How does something like this sound?

Reply:
+1

Reply:
I want to show how to do a fan-in. I don't think the rewording reflects that.